dags/utils/dag_configuration.py

```diff
@@ -16,6 +19,22 @@ def get_default_arguments():
     return default_arguments
 
 
+def get_dags_configuration():
+    dags_config = {
+        "osm": {
+            "roads": {"transform": default_transform},
+            "airports": {"transform": default_transform},
+            "rail": {"transform": default_transform},
+            "lakes": {"transform": default_transform},
+            "places": {"transform": default_transform},
+            "rivers": {"transform": default_transform},
+            "seaports": {"transform": default_transform},
+            "seas": {"transform": default_transform},
+        }
+    }
+    return dags_config
+
+
 def get_catchup():
```

Comment:
A docstring would be helpful here. I had to check what "catchup" meant in this context.